packages/sdk-react/src/components/auth/OtpVerification.tsx

```diff
@@ -36,7 +36,7 @@ const OtpVerification: React.FC<OtpVerificationProps> = ({
   sessionLengthSeconds,
   onValidateSuccess,
   onResendCode,
-  numBoxes,
+  numBoxes = 6, // default to 6 boxes
```

Comment:
IIRC, the default as of some recent updates was 9 boxes/digits for added security (which is why the `otp` component defaults to 9 boxes). Are we going back to 6?